extensions/blocks/button/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { button as icon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import attributes from './attributes';
+import edit from './edit';
+import save from './save';
+import { supportsCollections } from '../../shared/block-category';
+
+export const name = 'button';
+
+export const settings = {
+	title: __( 'Button', 'jetpack' ),
+	icon,
+	category: supportsCollections() ? 'grow' : 'jetpack',
```

Comment:
Since this block doesn't show up in the inserter, this doesn't matter much.
Though, I've updated this to `layout`, which is the Core Button category.